pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -120,16 +120,16 @@ func Apply(target *deployments.Target) error {
 			}
 			if nodeVersionInfoUpdate.HasMajorOrMinorUpdate() {
 				err = target.Apply(deployments.KubernetesBaseOSConfiguration{
-					KubeadmVersion:    nodeVersionInfoUpdate.Update.APIServerVersion.String(),
-					KubernetesVersion: nodeVersionInfoUpdate.Current.APIServerVersion.String(),
+					UpdatedVersion: nodeVersionInfoUpdate.Update.APIServerVersion.String(),
+					CurrentVersion: nodeVersionInfoUpdate.Current.APIServerVersion.String(),
 				}, "kubernetes.install-intermediate-node-pattern")
 				if err != nil {
 					return err
 				}
 			} else {
 				err = target.Apply(deployments.KubernetesBaseOSConfiguration{
-					KubeadmVersion:    nodeVersionInfoUpdate.Update.APIServerVersion.String(),
-					KubernetesVersion: nodeVersionInfoUpdate.Current.APIServerVersion.String(),
+					UpdatedVersion: nodeVersionInfoUpdate.Update.APIServerVersion.String(),
+					CurrentVersion: nodeVersionInfoUpdate.Current.APIServerVersion.String(),
 				}, "kubernetes.install-base-packages")
```

Comment:
```suggestion
				}, "kubernetes.install-intermediate-node-pattern")
```

Reply:
oh thanks, i missed that. i think we can then drop the whole `if nodeVersionInfoUpdate.HasMajorOrMinorUpdate()` and always install the intermediate pattern